apache/ports.conf.in

```diff
@@ -1,2 +1 @@
-Listen 80
```

Comment:
Probably it might make sense to have a short explanation of why this line needed to be deleted in the commit message?